packages/amplify_core_plugin_interface/pubspec.yaml

```diff
@@ -13,6 +13,8 @@ dependencies:
   amplify_storage_plugin_interface: ^0.0.1-dev.3
   amplify_auth_plugin_interface: ^0.0.1-dev.3
   amplify_analytics_plugin_interface: ^0.0.1-dev.3
+  amplify_datastore_plugin_interface:
+    path: ../amplify_datastore_plugin_interface
```

Comment:
Need to fix this local path, as before. I can see doing this as a smaller, subsequent PR after this one for testing purposes, I suppose.